cpp/daal/include/services/internal/x86_64/x86_64_kernel_defines.h

```diff
@@ -33,7 +33,6 @@
     #define DAAL_KERNEL_SSE2_CONTAINER(ContainerTemplate, ...)
     #define DAAL_KERNEL_SSE2_CONTAINER1(ContainerTemplate, ...)
     #define DAAL_KERNEL_SSE2_CONTAINER_CASE(ContainerTemplate, ...)
-    #define DAAL_KERNEL_SSE2_CONTAINER_CASE_SYCL(ContainerTemplate, ...)
```

Comment:
might be dumb question, but would cpu dispatching keep working for oneDAL SYCL interfaces?  it should. 

So in general comment - i'm not sure that all DAAL + SYCL combinations should be removed.

Reply:
oneDAL CPU/GPU dispatching is done here https://github.com/oneapi-src/oneDAL/blob/main/cpp/oneapi/dal/backend/dispatcher.hpp#L136

All the GPU-related code within cpp/daal folder does not affect oneDAL API; it only affects DAAL API.
My understanding is that we are going to remove GPU support from DAAL API at all, so removing those macros should be Ok.